internal/kv/etcd/embed_etcd_kv.go

```diff
@@ -38,6 +38,11 @@ import (
 // implementation assertion
 var _ kv.MetaKv = (*EmbedEtcdKV)(nil)
 
+const (
+	defaultRetryCount    = 3
```

Comment:
I'm wondering if it's better to have a retry library, which allows configuration for:
- max retry count
- max retry timeout
- which error to retry (based a unified error status code)
- enable retry with jitters

Reply:
we do have it with retry.go

Reply:
@xiaofan-luan @dentiny 
shall I make the necessary changes and raise a PR?
@amaypatil can you please take this up with me?

Reply:
@charleskakumanu sure, feel free to do sure